MANIFEST.in

```diff
@@ -1,2 +1,6 @@
 prune common
 include LICENSE
+graft tests
+graft docs
+global-exclude __pycache__
+global-exclude *.py[co]
```

Comment:
```suggestion
global-exclude *.py[cod]
```

From https://packaging.python.org/guides/using-manifest-in/#manifest-in-commands :smile:

Reply:
Thanks, I'll use that next time I need to make a similar change!